### PR TITLE
Topic/wglick/asdplng78

### PR DESCRIPTION
**Pull request by billglick**
Add puppet-profile_audit module tag v0.1.0 to control-repo

### CODE REVIEW

**Comment by billglick** [comment]
This is currently being tested on the following hosts:

- glick-client01
- glick-pup

File: Puppetfile [hunk 1]
```diff
@@ -10,9 +10,10 @@ mod 'herculesteam/augeasproviders_ssh', commit: 'e4eee3726d0472cba1d2d66a2d09031
 mod 'inkblot/ipcalc', '2.2.0'
 # mod 'MiamiOH-pam_access', '1.0.2'
 mod 'ncsa/pam_access', tag: 'v1.0.3', git: 'https://github.com/ncsa/puppet-pam_access'
-mod 'ncsa/profile_additional_packages', branch: 'v0.2.0', git: 'https://github.com/ncsa/puppet-profile_additional_packages'
```
Comment (billglick) [inline Puppetfile:13]: This was set wrong in the previous PR to control-repo, but now fixed.

File: data/common.yaml [hunk 2]
```diff
@@ -34,6 +34,8 @@ profile_allow_ssh_from_bastion::groups:
   - org_asd
   - org_irst
 
+profile_email::root_mail_target: "devnull@ncsa.illinois.edu"
```
Comment (billglick) [inline data/common.yaml:37]: I think this was overlooked when ncsa/profile_email was added to the control-repo.

Comment (andylytical) [reply data/common.yaml:37]: This is already in the profile. Why here with the exact same value?

File: Puppetfile (hunk 1 above)
Comment (andylytical) [reply Puppetfile:13]: LGTM

File: data/common.yaml (hunk 2 above)
Comment (billglick) [reply data/common.yaml:37]: I bet it is left over from before profile_email was completed. I'll strip it out of control-repo and retest.

**Review by andylytical — APPROVED** [review]
Looks good Bill!

**Pull request merged**